packages/@react-aria/overlays/docs/useOverlayTrigger.mdx

```diff
@@ -0,0 +1,222 @@
+```
+
+# useOverlayTrigger
+
+<p>{docs.exports.useOverlayTrigger.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['useOverlayTrigger']}
+  sourceData={[
+    {type: 'W3C', url: 'https://www.w3.org/TR/wai-aria-practices/#dialog_modal'}
+  ]} />
+
+## API
+
+<FunctionAPI function={docs.exports.useOverlayTrigger} links={docs.links} />
+
+## Features
+
+There is no built in way to create popovers or other types of overlays in 
+HTML. <TypeLink links={docs.links} type={docs.exports.useOverlayTrigger} />,
+combined with <TypeLink links={docs.links} type={docs.exports.useOverlayPosition} />,
+helps achieve accessible overlays that can be styled as needed.
+
+**Note: `useOverlayTrigger` only handles the overlay itself. It should be combined 
+with [useDialog](useDialog.html) to create fully accessible popovers. For menus,
+see [useMenuTrigger](useMenuTrigger.html), which builds on `useOverlayTrigger` and provides
+additional functionality specific to menus.**
+
+* Exposes overlay trigger and connects trigger to overlay with ARIA
+* Positions the overlay relative to the trigger when combined with <TypeLink links={docs.links} type={docs.exports.useOverlayPosition} />
+* Hides content behind the overlay from screen readers when combined with <TypeLink links={docs.links} type={docs.exports.useModal} />
+* Handles closing the overlay when interacting outside and pressing the Escape key, when combined with <TypeLink links={docs.links} type={docs.exports.useOverlay} />
+
+## Anatomy
+
+An overlay trigger consists of a trigger element (e.g. button) and an overlay
+(e.g. popover, menu, listbox, etc.). `useOverlayTrigger` handles exposing the
+trigger and overlay to assistive technology with ARIA. It should be combined
+with <TypeLink links={docs.links} type={docs.exports.useOverlay} /> to handle
+closing the overlay, <TypeLink links={docs.links} type={docs.exports.useModal} />
+to handle hiding content behind the overlay from screen readers, and optionally
+with <TypeLink links={docs.links} type={docs.exports.useOverlayPosition} /> to
+handle positioning the overlay relative to the trigger.
+
+`useOverlayTrigger` returns props that you should spread onto the trigger element and 
+the overlay container element:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useOverlayTrigger.return.id].properties} />
+</TypeContext.Provider>
+
+## Example
+
+This example shows how to build a typical popover overlay that is positioned relative to
+a trigger button. The content of the popover is a dialog, built 
+with <TypeLink links={dialogDocs.links} type={dialogDocs.exports.useDialog} />.
+
+The popover can be closed by clicking or interacting outside the popover, or by pressing the Escape key.
+This is handled by <TypeLink links={docs.links} type={docs.exports.useOverlay} />.
+Focus is restored back to the button that triggered the popover it when it closes by a
```

Comment:
Maybe "When the popover is closed, focus is restored back to its trigger button by a FocusScope."